Mage/src/main/java/mage/abilities/effects/common/ExileWithTimeCountersEffect.java

```diff
@@ -0,0 +1,52 @@
+package mage.abilities.effects.common;
+
+import mage.abilities.Ability;
+import mage.abilities.effects.OneShotEffect;
+import mage.abilities.keyword.SuspendAbility;
+import mage.cards.Card;
+import mage.constants.Outcome;
+import mage.counters.CounterType;
+import mage.game.Game;
+import mage.players.Player;
+import mage.util.CardUtil;
+
+import java.util.UUID;
+
+/**
+ * @author PurpleCrowbar
+ */
+public class ExileWithTimeCountersEffect extends OneShotEffect {
```

Comment:
I think I'd prefer the name `ExileSpellWithTimeCountersEffect`

Reply:
Done